advanced/Scripts/webpage.sh

```diff
@@ -226,7 +232,26 @@ trust-anchor=.,20326,8,2,E06D44B80B8F1D39A95C0B0D7C65D08458E880409BBC68345710423
         REV_SERVER_TARGET="${CONDITIONAL_FORWARDING_IP}"
         add_setting "REV_SERVER_TARGET" "${REV_SERVER_TARGET}"
 
-        REV_SERVER_CIDR="${CONDITIONAL_FORWARDING_REVERSE}"
+        #Convert CONDITIONAL_FORWARDING_REVERSE if necessary e.g:
+        #          1.1.168.192.in-addr.arpa to 192.168.1.1/32
+        #          1.168.192.in-addr.arpa to 192.168.1.0/24
+        #          168.192.in-addr.arpa to 192.168.0.0/16
+        #          192.in-addr.arpa to 192.0.0.0/8
+        search="in-addr.arpa"
+        if [[ "$CONDITIONAL_FORWARDING_REVERSE" == *"$search" ]];then
+            arrRev=("${CONDITIONAL_FORWARDING_REVERSE//./ }")        
+            case ${#arrRev[@]} in 
+                6   )   REV_SERVER_CIDR="${arrRev[3]}.${arrRev[2]}.${arrRev[1]}.${arrRev[0]}/32";;
+                5   )   REV_SERVER_CIDR="${arrRev[2]}.${arrRev[1]}.${arrRev[0]}.0/24";;
+                4   )   REV_SERVER_CIDR="${arrRev[1]}.${arrRev[0]}.0.0/16";;
+                3   )   REV_SERVER_CIDR="${arrRev[0]}.0.0.0/8";; 
+            esac
+        else
+          # Set REV_SERVER_CIDR to whatever value it was set to
+          REV_SERVER_CIDR="$CONDITIONAL_FORWARDING_REVERSE"
+        fi
+        
+        # If REV_SERVER_CIDR is not converted by the above, then use the REV_SERVER_TARGET variable to derive it
         if [ -z "${REV_SERVER_CIDR}" ]; then
```

Comment:
So this condition really is `if $CONDITIONAL_FORWARDING_REVERSE is empty`, right? Because we set 
``` bash
REV_SERVER_CIDR="$CONDITIONAL_FORWARDING_REVERSE"
```
above if `in-addr.arpa` is not found

Reply:
Yeah, I guess that's another way of looking at it.

I'm in two minds about the above else block if `in-addr.arpa` is not found.

1) it's not `in-addr.arpa`, but it could be, say `192.168.0.0/16`. If we do not set `REV_SERVER_CIDR` at this point, then we will assume a `/24` down the bottom.
2) it could be a garbage value, in which case `REV_SERVER_CIDR` will be broken anyway

Reply:
It's fine. Option 1 is covered, option 2 (garbage value) isn't anything we can really handle. The user will see FTL failing to start and will investigate, finding his error (or asking for help). This seems fine.